src/mlpack/tests/q_learning_test.cpp

```diff
@@ -438,17 +436,8 @@ BOOST_AUTO_TEST_CASE(DoublePoleCartWithDQN)
       double episodeReturn = agent.Episode();
       episodes += 1;
 
-      if (episodes > 1000)
-      {
-        Log::Debug << "Multiple Pole Cart with DQN failed." << std::endl;
-        converged = false;
-        break;
-      }
-
-      if (episodeReturn >= 380)
+      if (episodeReturn >= 280)
         successive++;
-      else if (successive)
-        successive = 0;
 
       // If the model can solve the environment in two consecutive trials this
```

Comment:
I guess this comment is a little inaccurate---it seems to me like it no longer needs to be two consecutive trials.